app/models/mailing_list/wizard.rb

```diff
@@ -59,10 +59,28 @@ def exchange_magic_link_token(token)
   private
 
     def add_member_to_mailing_list
-      request = GetIntoTeachingApiClient::MailingListAddMember.new(export_camelized_hash)
+      request = GetIntoTeachingApiClient::MailingListAddMember.new(construct_export)
       api = GetIntoTeachingApiClient::MailingListApi.new
       Rails.logger.info("MailingList::Wizard#add_mailing_list_member: #{AttributeFilter.filtered_json(request)}")
       api.add_mailing_list_member(request)
     end
+
+    def construct_export
+      export = export_camelized_hash
+
+      show_welcome_guide = ApplicationController.helpers.show_welcome_guide?(
```

Comment:
The `show_welcome_guide?` method should perhaps be pulled out into its own module rather than pulling in a view helper here, but I'm not sure if its worth it 🤷‍♂️